pandas/_libs/tslib.pyx

```diff
@@ -3892,8 +3892,9 @@ for field in fields:
 _nat_methods = ['date', 'now', 'replace', 'to_pydatetime',
                 'today', 'round', 'floor', 'ceil', 'tz_convert',
                 'tz_localize']
-_nan_methods = ['weekday', 'isoweekday', 'total_seconds']
-_implemented_methods = ['to_datetime', 'to_datetime64', 'isoformat']
+_nan_methods = ['weekday', 'isoweekday']
```

Comment:
This blows away the doc-string. So I would rather leave this as-is and simply remove the ``total_seconds`` method from ``NaT`` class; accomplishing the same thing more in-line with current code.

Reply:
Fair enough.  I defaulted to leaving the `def total_seconds` in place because it had a GH issue attached to it.  Can change, or can just pin on the correct docstring up where the method is defined.

Reply:
see my comments, its would be faster just to make the change.